framework/decode/metadata_json_consumer.h

```diff
@@ -314,6 +315,40 @@ class MetadataJsonConsumer : public Base
         WriteBlockEnd();
     }
 
+    virtual void ProcessBuildVulkanAccelerationStructuresMetaCommand(
+        format::HandleId                                                           device_id,
+        uint32_t                                                                   info_count,
+        StructPointerDecoder<Decoded_VkAccelerationStructureBuildGeometryInfoKHR>* geometry_infos,
+        StructPointerDecoder<Decoded_VkAccelerationStructureBuildRangeInfoKHR*>*   range_infos) override
+    {
+        auto&              jdata        = WriteMetaCommandStart("BuildVulkanAccelerationStructuresMetaCommand");
+        const JsonOptions& json_options = GetJsonOptions();
+        FieldToJson(jdata["infoCount"], info_count, json_options);
+        FieldToJson(jdata["pInfos"], geometry_infos, json_options);
+        FieldToJson(jdata["ppBuildRangeInfos"], range_infos, json_options);
+        WriteBlockEnd();
+    }
+
+    void ProcessCopyVulkanAccelerationStructuresMetaCommand(
+        format::HandleId                                                  device_id,
+        StructPointerDecoder<Decoded_VkCopyAccelerationStructureInfoKHR>* copy_infos) override
+    {
+        auto&              jdata        = WriteMetaCommandStart("CopyVulkanAccelerationStructureMetaCommand");
+        const JsonOptions& json_options = GetJsonOptions();
+        FieldToJson(jdata["pInfo"], copy_infos, json_options);
+        WriteBlockEnd();
+    }
+
+    virtual void ProcessVulkanAccelerationStructuresWritePropertiesMetaCommand(
+        format::HandleId device_id, VkQueryType query_type, format::HandleId acceleration_structure_id) override
+    {
+        auto&              jdata        = WriteMetaCommandStart("BuildVulkanAccelerationStructuresMetaCommand");
```

Comment:
Should this be `WriteMetaCommandStart("VulkanWriteAccelerationStructuresPropertiesCommand");`?

Reply:
For this and the other two changes to the meta Json consumer, I extracted https://github.com/LunarG/gfxreconstruct/pull/2585 and fixed up the consumer names, which were not consistent with format.h.  If you want to drop out the AS commits from this PR, you could resolve these comments and then review 2585.